environment/src/main/java/com/sequenceiq/environment/environment/flow/start/handler/StartFreeIpaHandler.java

```diff
@@ -39,12 +43,16 @@ public String selector() {
     public void accept(Event<EnvironmentStartDto> environmentStartDtoEvent) {
         EnvironmentDto environmentDto = environmentStartDtoEvent.getData().getEnvironmentDto();
         try {
-            freeIpaService.describe(environmentDto.getResourceCrn()).ifPresent(freeIpa -> {
-                if (freeIpa.getStatus() != null && !freeIpa.getStatus().isStartable()) {
+            freeIpaService.describe(environmentDto.getResourceCrn()).ifPresentOrElse(freeIpa -> {
+                if (freeIpa.getStatus() != null && (freeIpa.getStatus().isAvailable() || freeIpa.getStatus().isStartInProgressPhase())) {
+                    LOGGER.info("FreeIPA is already {}, so it won't started", freeIpa.getStatus());
```

Comment:
Grammar: `it won't be started` I think it should be something like `Start has already been trigger continuing without new start trigger.